ui/src/variables/components/VariableDropdown.tsx

```diff
@@ -43,6 +43,13 @@ class VariableDropdown extends PureComponent<Props> {
     const dropdownStatus =
       values.length === 0 ? ComponentStatus.Disabled : ComponentStatus.Default
 
+    const widthLength = Math.max(
+      140,
+      values.reduce(function(a, b) {
+        return a.length > b.length ? a : b
+      }, '').length * 8
```

Comment:
how did you choose the number eight?

Reply:
Im not going to lie, it looked the best in the UI. I tried a few numbers, and 8 looked ideal for the px

Reply:
is the intent to find out what the max width of the longest variable _could_ be?

Reply:
So the intent is find the longest variable, then take its length and determine how much width in px would be needed to display it